requirements/bddtests.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile requirements/bddtests.in
+#    pip-compile --allow-unsafe requirements/bddtests.in
```

Comment:
We're probably going to have some push and pull with dependabot with things like this

Reply:
I've actually tested this with Dependabot in a private repo and I think it's going to be fine. I think Dependabot might actually _read the `pip-compile` arguments from these comments_ that `pip-compile` puts into the requirements.txt files :open_mouth: In any case in my test repo Dependabot didn't fight me on this. If it does then we may need to remove `--allow-unsafe`